include/fiction/io/write_operational_domain.hpp

```diff
@@ -22,14 +24,34 @@ namespace fiction
  */
 struct write_operational_domain_params
 {
+    /**
+     * Mode selector for writing samples to file.
+     */
+    enum class sample_writing_mode : uint8_t
+    {
+        /**
+         * Write all samples, including non-operational ones. This may lead to large file sizes.
+         */
+        ALL_SAMPLES,
+        /**
+         * Write operational samples only. This can drastically reduce file size and help with visibility in 3D plots.
+         */
+        OPERATIONAL_ONLY
+    };
     /**
      * The tag used to represent the operational value of a parameter set.
      */
-    std::string_view operational_tag = "operational";
+    std::string_view operational_tag = "1";
     /**
      * The tag used to represent the non-operational value of a parameter set.
      */
-    std::string_view non_operational_tag = "non-operational";
+    std::string_view non_operational_tag = "0";
+    /**
+     * Whether to write non-operational samples to the CSV file. If set to `OPERATIONAL_ONLY`, operational samples are
+     * written exclusively. This yields a significantly smaller CSV file. It is recommended to set this option for 3D
+     * plots because the non-operational samples would shadow the operational samples anyway.
+     */
+    sample_writing_mode writing_mode = sample_writing_mode::ALL_SAMPLES;
```

Comment:
@marcelwa Do we want to provide this option in _pyfiction_? If so, we should adjust the bindings accordingly.

Reply:
You're right! Done.